tests/genarg/dune

```diff
@@ -63,3 +63,8 @@
  (name runtest)
  (deps (:input symmetry.v))
  (action (ignore-stdout (run sercomp --exn_on_opaque %{input}))))
+
+(alias
+ (name runtest)
+ (deps (:input fix.v))
+ (action (ignore-stdout (run sercomp --exn_on_opaque %{input}))))
```

Comment:
We are keeping the tests in alphabetical order as to have a more efficient human-lookup procedure :)